sdk/src/main/scala/com/horizen/block/SidechainBlockHeader.scala

```diff
@@ -47,6 +47,7 @@ case class SidechainBlockHeader(
   lazy val messageToSign: Array[Byte] = {
     Bytes.concat(
       idToBytes(parentId),
+      Array[Byte]{version},
```

Comment:
Do the same change for the `BoxTransaction messageToSign` method.

Reply:
Done.